municipal_finance/middleware.py

```diff
@@ -38,7 +38,7 @@ def __call__(self, request):
     
     def process_request(self, request):
         site = get_current_site(request)
-        if site.name == 'Scorecard':
+        if site.name == 'Scorecard' or site.name == 'example.com':
```

Comment:
could you see which site objects exist in the DB at this point? Our migrations should be creating the right ones. Perhaps we need the test to edit the relevant object to have the netloc (hostname and port) used in the test?

Reply:
For some reason one of the selenium tests reverts to the default example.com domain. I removed the code changes to prod and update the domain in the test